File: app/src/main/java/com/mifos/apache/fineract/data/models/loan/TermRange.java
package com.mifos.apache.fineract.data.models.loan;

/**
 * @author Rajan Maurya
 *         On 12/07/17.
 */
public class TermRange {

    private String temporalUnit;
    private Double maximum;

    public TermRange(String temporalUnit, Double maximum) {
        this.temporalUnit = temporalUnit;
        this.maximum = maximum;
    }

    public String getTemporalUnit() {
        return temporalUnit;
    }

    public void setTemporalUnit(String temporalUnit) {
        this.temporalUnit = temporalUnit;
    }

    public Double getMaximum() {
        return maximum;
    }

    public void setMaximum(Double maximum) {
        this.maximum = maximum;
    }
}

File: app/src/main/java/org/apache/fineract/data/models/loan/TermRange.kt
package org.apache.fineract.data.models.loan

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * @author Rajan Maurya
 * On 12/07/17.
 */
@Parcelize
data class TermRange(
    @SerializedName("temporalUnit") var temporalUnit: String?,
    @SerializedName("maximum") var maximum: Double?
) : Parcelable
